workbench_utils.py

```diff
@@ -11582,6 +11595,33 @@ def get_config_file_identifier(config):
 
     return config_file_id
 
+def get_config_file_identifier_shortened(config):
+    """Gets a unique identifier of the current config file. Used in names of temp files, etc."""
+    """Parameters
+        ----------
+        config : dict
+            The configuration settings defined by workbench_config.get_config().
+        Returns
+        -------
+        string
+            A string based on just the config file's name (without path or extension).
+    """
+    global _session_temp_identifier
+    
+    # Extract just the filename without path and extension
+    config_file_path = config["current_config_file_path"]
+    config_file_name = os.path.basename(config_file_path)
+    config_file_id = os.path.splitext(config_file_name)[0]
+
+    # Create session-specific identifier once per session using process ID for uniqueness
+    if _session_temp_identifier is None:
+        with tempfile.NamedTemporaryFile(delete=False, suffix=f"_{config_file_id}", prefix=f"pid{os.getpid()}_") as temp:
+            _session_temp_identifier = os.path.basename(temp.name)
+            # Clean up the temporary file since we only need the name
+            os.unlink(temp.name)
```

Comment:
If you remove the `delete=False` from the `NamedTemporaryFile` above, it will automatically be deleted when you leave the `with` statement.